include/despf.inc.sh

```diff
@@ -110,7 +110,13 @@ parsepf() {
   then
     myns=$(findns $host 2>/dev/null)
   else
-    myns=$(sed -n 's/^nameserver \([\.:0-9a-f]*\)/\1/p' /etc/resolv.conf)
+    if 
+      test $DNS_SERVER
```

Comment:
Fair enough. This is really the first time I see `test` being used without parameters. Always learning. I see it is even defined in the POSIX - http://pubs.opengroup.org/onlinepubs/9699919799/

> string
> . . . . True if the string string is not the null string; otherwise, false.